userguide/plugins.rst

```diff
@@ -159,6 +159,10 @@ When a newer version of a plugin becomes available in the official
 repository, update the plugin jail by navigating to the
 :menuselection:`Jails` page and clicking |ui-options| and
 :guilabel:`Update`.
+
+.. note:: Plugins are updated using :samp:`iocage update {PLUGIN}`
```

Comment:
Wait... this is not clear on several things.  What is the difference between an "update" and an "upgrade"?  Is this aimed at the end user or the plugin developer?  Does the end user have to use the command line to update or upgrade plugins?